docker/ubuntu/Dockerfile

```diff
@@ -0,0 +1,87 @@
+# --------------------------------------------------------------------
+# Copyright (c) 2020, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------
+
+# set base Docker image to AdoptOpenJDK Ubuntu Docker image
+FROM adoptopenjdk:8u222-b10-jdk-hotspot
+LABEL maintainer="dev@wso2.org"
+
+ENV LANG=C.UTF-8
+
+# set Docker image build arguments
+# build arguments for user/group configurations
+ARG USER=ballerina
+ARG USER_ID=802
+ARG USER_GROUP=ballerina
+ARG USER_GROUP_ID=802
+ARG USER_HOME=/home/${USER}
+
+# build arguments for WSO2 product installation
+ARG MGW_SERVER_NAME=wso2am-micro-gw
+ARG MGW_SERVER_VERSION=3.0.2
```

Comment:
3.0.x versions are maintained in 3.0.x branch. Can you send this PR to 3.0.x branch?